src/server/implementation/mutations/retrieveAccountWithCredentials.ts

```diff
@@ -60,8 +60,6 @@ export async function retrieveAccountWithCredentialsImpl(
   }
   return {
     account: existingAccount,
-    // TODO: Ian removed this
-    user: (await ctx.db.get(existingAccount.userId))!,
```

Comment:
I don't remember why this was added back. I thought it wasn't needed beyond getting the userId, but there's a decent amount of untyped code..